src/cli/cli.cc

```diff
@@ -383,6 +383,18 @@ void runIOSSimulator (const fs::path& path, Map& settings) {
     exit(rInstallApp.exitCode);
   }
 
+  log("launching the child logger process");
```

Comment:
maybe we output this for debug only?

Reply:
Yes, I don't think we need it. The purpose of this PR is to save a draft of how to enable logging. Currently, streaming doesn't work for me, I only see a first message from `log stream` about starting to listen to the log

Reply:
yep log stream only works for local stores